src/utils/fetchSetIndexComponents.ts

```diff
@@ -0,0 +1,32 @@
+import { camelCase } from 'lodash'
+
+const proxyPrefixURL = 'https://cors-anywhere.herokuapp.com/'
```

Comment:
we'll probably end up removing this, but i'll handle it after testing the deploy

Reply:
I tested fetch from the console on indexcoop.com and I also got a CORS from that domain so we still need it when deployed.
My understanding is that using this API endpoint is temporary until the subgraph is done and then this data will be available with minimal calculations on frontend right?

Reply:
Ok we just ended up changing the set-core endpoint to allow CORS on all GET requests.